### DISCUSSION TITLE
Issue with running MRtrix GUI on Neurodesktop

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#124] [Q&A] [answered]
Hello Neurodeskers,

I'm cross-posting this discussion from the MRtrix's forum. I tried to help as much as I could, but please pitch in if you can help further.
https://community.mrtrix.org/t/mrtrix3-docker-gui-issue/5273
If you don't want to open a user on the MRtrix's forum, just post you feedback here, and I'll link it to the MRtrix forum.

Thanks,
Oren

**Comment by stebo85** [comment, marked as answer]
There is nothing really to help? All seems to work now?
